src/domain/contracts/decoders/__tests__/safe-decoder.helper.spec.ts

```diff
@@ -1,12 +1,12 @@
-import { SafeDecoder } from '@/domain/contracts/contracts/safe-decoder.helper';
+import { SafeDecoder } from '@/domain/contracts/decoders/safe-decoder.helper';
 import {
   addOwnerWithThresholdEncoder,
   changeThresholdEncoder,
   execTransactionEncoder,
   removeOwnerEncoder,
   setupEncoder,
   swapOwnerEncoder,
-} from '@/domain/contracts/contracts/__tests__/safe-encoder.builder';
+} from '@/domain/contracts/encoders/safe-encoder.builder';
```

Comment:
Related to my previous comment: not sure if this should be kept under test folders or you maybe you are foreseeing a purpose apart from testing.

Reply:
I moved the encoders in ca9b06462159ce6724ffcc41b3e0216e20d63932 as per [my previous response](https://github.com/safe-global/safe-client-gateway/pull/1174#discussion_r1497351085).